### DISCUSSION TITLE
RaspberryPi with RaZberry

### DISCUSSION THREAD

**Discussion by excessnet** [zwave-js/node-red-contrib-zwave-js#64] [Q&A] [answered]
Hello,

I got Node-Red running on a Raspberry (3 b+ I think) and I was wondering if I could plug my **RaZberry V2** on it so I can use this node to connect and manage the "GE 14285 Z-Wave Plus 40-Amp Indoor/Outdoor Metal Box Smart Switch" ( https://www.amazon.ca/gp/product/B00YTCZZF0 ) and maybe others things along the way.

Thanks! :)

**Comment by marcus-j-davies** [comment, marked as answer]
Hi,

Yes it should - the **RaZberry V2** uses the Z-Wave Serial API, so it should work.
this module doesn't care what you use, as long as the Z-Wave Module offers the Serial API - most do.

**Reply by excessnet** [reply]
cool thanks a lot! I'll buy the relay try it then! :)

**Reply by marcus-j-davies** [reply]
The switch is supported, buy the backend library (that this module uses, and one I am involved with)
https://devices.zwave-js.io/?jumpTo=0x0063:0x4f44:0x3032:0.0

I would first make sure, the **RaZberry V2** does work, before making any significant purchase.

as long as you get "All Nods Ready" on the node in your flow - you good to go.

**Reply by excessnet** [reply]
okay I'll try to connect the RaZberry V2 to the Raspberry et see if it's working first.

nice to know the switch is supported...  thought it was working with everything "z-wave"! I'll look into that list next time! :)

**Comment by marcus-j-davies** [comment]
Choose wisely,

the switch  you want, seems like its trouble (down to the manufacture fixing it) either with a firmware update or otherwise.

https://github.com/zwave-js/node-zwave-js/issues/2196

**Reply by excessnet** [reply]
oops, I'll look for something else then! :)

**Reply by marcus-j-davies** [reply]
no harm in trying - they may have fixed it with a FW update - plus you got amazon returns
